querybook/server/lib/query_executor/base_executor.py

```diff
@@ -528,6 +528,16 @@ def __init__(
         self._client = None
         self._cursor = None
 
+        with DBSession() as session:
+            query_execution_metadata = (
+                qe_logic.get_query_execution_metadata_by_execution_id(
+                    self._query_execution_id, session=session
+                ).execution_metadata
+            )
+            self._api_access_token = query_execution_metadata.get(
```

Comment:
rename to used_api_token or some other boolean name

api_access_token was used as a str to represent the actual token, let's not reuse the same name for a boolean flag?